src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -68,6 +68,11 @@ private abstract class AbstractTableDataSourceFindUsagesContext :
             /// </summary>
             private bool _cleared;
 
+            /// <summary>
+            /// Message we show if we find no definitions.  Consumers of the streaming presenter can set their own title.
+            /// </summary>
+            protected string NoDefinitionsFoundMessage = ServicesVSResources.Search_found_no_results;
```

Comment:
we used to hardcode this.  now we just support the contract we always had before (but this type ignored).